.github/workflows/on-pull-request.yml

```diff
@@ -38,6 +38,22 @@ jobs:
         run: brm validate
         working-directory: ${{ needs.get-module-to-validate.outputs.module_dir }}
 
+  shellcheck:
+    name: runner / shellcheck
```

Comment:
For sake of consistency, do you mind removing the job name?

Reply:
```suggestion
    name: runner
```